README.md

```diff
@@ -99,20 +134,40 @@ Manage traditional [variants] of a configuration - like
 _development_, _staging_ and _production_ - using
 [overlays] that modify a common [base].
 
-![overlay image][imageOverlay]
+```
+
+overlay: kustomization + patches + more resources
+
+kustomization.yaml                              replica_count.yaml                     cpu_count.yaml
++---------------------------------------+       +-------------------------------+      +------------------------------------------+
+| commonLabels:                         |       | apiVersion: apps/v1           |      | apiVersion: apps/v1                      |
+|   app: myapp                          |       | kind: Deployment              |      | kind: Deployment                         |
```

Comment:
```suggestion
|   variant: prod                          |       | kind: Deployment              |      | kind: Deployment                         |
```

(it already has the 'app' label from the base)

Reply:
Any particular reason why you removed the commonAnnotations and namePrefix fields that were in the original example?

Reply:
@KnVerey , removed it to keep things simple. Should we add it back?